DavidGonzalezZumaya/Collections/src/RansomNote/RansomeNote.java

```diff
@@ -0,0 +1,48 @@
+package RansomNote;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RansomeNote {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+
+        Map<String, Integer> magazineMap = createMapOfMagazine(magazine);
+
+        if(validateMessage(magazineMap, note)){
+            System.out.print("Yes");
+        }
+        else{
+            System.out.print("No");
+        }
+    }
+
+    public static Map<String, Integer> createMapOfMagazine(List<String> magazine ){
+        Map<String, Integer> MagazineWords = new HashMap<String, Integer>();
```

Comment:
It is recommended that the name of the collections and maps start with singular name and them the name of the interface, example: numberList

Reply:
LowerCamelCase